examples/main/main.cpp

```diff
@@ -781,7 +787,9 @@ int main(int argc, char ** argv) {
                         n_consumed = embd_inp.size();
                         embd_inp.insert(embd_inp.end(), inp_pfx.begin(), inp_pfx.end());
                     }
-
+                    if (params.escape) {
+                        process_escapes(buffer);
+                    }
```

Comment:
Do we not also want to keep this to escape the new input?

Reply:
@vvhg1
That's the current behavior, user input in interactive mode tokenizes "\n" as `\`,`n`

While enabling escapes for interactive user input would allow "multiline" input via "\n", it would prevent user from writing "\n" literally, making it impossible to do things like copy-pasting code snippets etc.

Reply:
@staviq I see your point. What I don't yet understand is, why would we want to treat the initial prompt differently than subsequential prompts?

Reply:
Ah, wait - I think I was wrong. Let me check this again later, but we might want to keep this escape

Reply:
Ok, I agree we should keep this escape as well

Reply:
ok, should all be ready to merge